proto/sdk/sdk.proto

```diff
@@ -131,7 +142,7 @@ message GameServer {
         Health health = 1;
 
         message Health {
-            bool disabled = 1;
+            bool disabled = 1 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {format: "boolean"}];
```

Comment:
Out of curiosity, what happens if you leave this off?

Reply:
If [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {format: "boolean"}] is not defined, the output of swagger.json is missing format: "boolean".
In grpc-gateway v2, "boolean" format isn't outputed by default.

This is specification by OpenAPI v2 scheme.
https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#data-types

Reply:
Got it! Thanks!